ee/tabby-ui/app/chat/page.tsx

```diff
@@ -222,11 +223,27 @@ export default function ChatPage() {
     }
   }, [server, client])
 
+  const [serverCapabilities, setServerCapabilities] = useState(
+    new Map<string, boolean>()
+  )
+
```

Comment:
instead of using a map, use explicit boolean field for ease of compiler check.

e.g
```
const supportsOnApplyInEditorV2 = useState(false);

...
```